### ISSUE TITLE
Upload keystore / Play-Store managed app-signing

### ISSUE THREAD

**Issue by hollmannlu** [PatilShreyas/bytemask#21]
Hey,
does this also work with apps, where app-signing is done via google play and there is only an upload keystore?
thank you and kind regards,
lukas

**Comment by Oleg-Beloy** [comment]
Yes, it works. However, beware of https://github.com/PatilShreyas/bytemask/issues/20 crash in production
